### ISSUE TITLE
overgang naar swing stories

### ISSUE THREAD

**Issue by Emilien-Dupont** [provinciesincijfers/PinC-taakopvolging#423]
bouwt verder op: https://vlbr.sharepoint.com/:w:/t/DA-Interprovincialewerking/EUuY4LBmfhpImWQa87Z4XnkBfslwkfrDmyp9VxH_bJvuaA 

we kunnen swing stories nu uittesten in de beta (sinds 20/11):
https://beta-provincies.incijfers.be/admin/mosaicstudio/theme?mosaicName=Mosaic2&id=cgdid8gjhakagip
update 27/11: onderaan in de mosaci van beta staan er nog 5 extra testrapporten

issues waar de centrumsteden al tegenaan liepen: 
https://github.com/provinciesincijfers/swing-stories/issues

documentatie op teams: https://vlbr.sharepoint.com/:f:/r/teams/DA-Interprovincialewerking/Gedeelde%20documenten/General/Overgang%20rapport%20naar%20stories?csf=1&web=1&e=whrijP

wordt verder opgenomen in de werkgroep swing stories

**Comment by Emilien-Dupont** [comment]
Overzicht van problemen waar de werkgroep swing stories tegenaan liep, met prioriteiten, staat op [teams](https://vlbr.sharepoint.com/:w:/r/teams/DA-Interprovincialewerking/Gedeelde%20documenten/General/Overgang%20rapport%20naar%20stories/Overzicht%20openstaande%20issues%20en%20moeilijkheden%20Stories.docx?d=w3010ab1870da42ffb6f6eb3c29646d38&csf=1&web=1&e=b3RbMY)

**Comment by Emilien-Dupont** [comment]
handigheidje door Stefan ontdekt, een pop-up in de tekst:
![image](https://github.com/provinciesincijfers/PinC-taakopvolging/assets/77432663/f343521b-4ab9-4b3d-9c53-f4fac1a079cc)

Hoe te gebruiken:
(1) De Glossary invullen: Als je klikt op ‘Mosaic Settings' van je story kan je daar een glossary ingeven. 
(2) Verwijzen naar de Glossary in de tekst zelf, via get-statements.

Achter de help-knop in de Studio:
The get-statement has the following form:
`{get: glossary-term; termcode: <termcode>; textterm: <textterm>}`
The <termcode> is a code defined in the Glossary tab of the Mosaic Settings.
<textterm> is optional. If supplied this text is used in the text. Else the term defined in the glossary will be used.
Get example
`{get: glossary-term; termcode: difficultterm; textterm: Difficult}`

**Comment by Emilien-Dupont** [comment]
Dat icoontje kunnen we zelf niet instellen. In de leeswijzer staat wel vermeld hoe gebruikers de data in de databank kunnen opvragen: 

Alle cijfers in dit rapport zijn ook raadpleegbaar via onze [Databank](https://provincies.incijfers.be/databank). Klik hiervoor op de figuur of kaart. Er opent zich een pop-up, daar heb je de mogelijkheid om de weergave van de presentatie te veranderen of om te openen in de databank.

Maar het is inderdaad heel verwarrend. Ik heb het ingeseind bij ABF met de vraag of er voor het iframe link icoon geen ander icoon kan gekozen worden.

**Comment by Emilien-Dupont** [comment]
antwoord van abf:
Dit is het standaard icoon voor het openen van een URL in een nieuwe tabblad/scherm. Deze is hetzelfde omdat in beide gevallen de link achter de presentatie wordt geopend. We gaan tegen de standaard in als we hiervan af gaan wijken, dus we laten het voorlopig zo.

**Comment by GeertBonte** [comment]
Naar aanleiding van het webinar over de SDG-story van deze voormiddag heb ik voor een aantal gemeenten de volledige story gedownload als pdf, of althans een poging daartoe gedaan. Ik stel vast dat niet lang niet altijd van een leien dakje loopt. Mijn pdf-bestanden voor Aalst en Leuven waren oké, maar in die voor Hove en Brugge ontbreken er een aantal kaartjes en een afsluitende grafiek! Met Antwerpen, Gent en Knokke-Heist kreeg ik meermaals een pdf met slechts één pagina, maar bij een latere poging kreeg ik wel een compleet bestand.

Het probleem met de ontbrekende kaarten en grafieken bij Hove en Brugge deed zich ook voor toen ik op een andere computer en op een ander moment probeerde. Daar gaat dus iets structureel fout.

Onderstaande afbeelding is geen schermafdruk van de website, maar komt uit het pdf-bestand:

![sdg_story_brugge](https://github.com/provinciesincijfers/PinC-taakopvolging/assets/101627698/45333a38-6c8c-4093-bfad-4253463e6748)

**Comment by Tine-VB** [comment]
Het lijkt inderdaad nogal willekeurig te zijn. Ook Stefan en ik krijgen het rapport van Hove niet in orde. Wel is het enkel bij de 'viewer presentations' die Stefan heeft moeten aanmaken omdat z-kaarten en profielstaven niet mogelijk zijn in Stories. Emilien heeft het toegevoegd aan het verzameldocument dat volgende week aan ABF wordt bezorgd. We vragen dus om dit toch mogelijk te maken in de Stories, en alleszins moet dit huidige probleem snel opgelost worden.

**Comment by Emilien-Dupont** [comment]
Ik heb het net ook ingeseind bij abf

**Comment by StefanJacques** [comment]
nagelezen door Limburg - opmerkingen per mail naar Hanne en Lieven
**ziet er zeer goed uit, collega's !!!**

**Comment by HanneODB-VLBR** [comment]
> wat puntje 2 betreft: voor de GSM indicatoren zou ik de 'hoger, lager, ....' zo veel mogelijk proberen te vermijden.

Inderdaad, nu ik het zelf nog eens opnieuw overlees is het eigenlijk echt niet ok om het zo te benoemen, zelfs mits de waarschuwing in de inleiding (die door velen niet zal worden gelezen :-)). Ik heb alle dergelijke verwijzingen verwijderd, en ook de tekst nog herschreven om beter aan te geven dat het om schattingen gaat ('rond de x %', 'ongeveer x %'), plus de percentages afgerond zodat er geen cijfers na de komma meer worden getoond (om het idee van exactheid weg te nemen).

**Comment by Emilien-Dupont** [comment]
Bijkomend puntje redactie: planning opmaken welke rapporten we wanneer omzetten naar stories in 2025

**Comment by Tine-VB** [comment]
Timing Stories van Vlaams-Brabant:
- ELZ: opstart in 2024, klaar tegen begin 2025
- Landbouw: opstart nog in 2024
- Arbeidsmarkt: opstart nog in 2024
- Onderwijs: data en tekst aan te passen, in de loop van 2025
- Detailhandel: er wordt elk jaar een heel nieuw rapport geschreven tegen mei, dus in principe Stories detailhandel tegen mei 2025.
- Demografie: loop van 2025
- Armoede: mei/juni 2025
- Cijfers opgelijst (economie en detailhandel): start afhankelijk van sjabloon voor de omzetting van cijferfiches naar Stories.

Omgevingsanalyse: Hoe? Wie?

**Comment by Tine-VB** [comment]
Stories mobiliteit: in orde voor publicatie.

**Comment by StefanJacques** [comment]
2024
Natuur – in verwerking op dit moment (september – half oktober)
Fossielvrij bouwen en wonen - NIEUW (half oktober - november)
Inclusieve arbeidsmarkt – NIEUW (december)
 
2025
Inclusieve arbeidsmarkt – NIEUW (januari)
Herwerking - Criminaliteit (februari) – dit rapport is rechttoe-rechtaan – geen kaarten, enkel staafgrafieken en 1 lijngrafiek
Herwerking – kerncijfers (februari) – dit rapport bestaat uit verwijzingen naar profielstaven
Herwerking – Ruimtegebruik (maart)
 
vanaf april
Klimaat – mitigatie
Klimaat – scenario’s
Klimaat – adaptatie
Klimaat - energiearmoede

Cijfers opgelijst (bevolking, huishoudens, mitigatie): start afhankelijk van sjabloon voor de omzetting van cijferfiches naar Stories.

**Comment by Emilien-Dupont** [comment]
Redactiegroep 19/09/24:
We roepen de werkgroep Stories zo snel mogelijk samen. Deze staat open voor iedereen.
Te bespreken: plaats hergebruiksvoorwaarden Stories. 

Mobiliteit mag online komen. 

TODO: planning rapporten opmaken (elk team).
TODO: navragen bij ABF wat de limiet is van stories (Emilien). 
Omgevingsanalyse: inhoudelijke revisie + omzetting naar Stories: geen prioriteit.

**Comment by Lievendr** [comment]
Antwerpen neemt storie Demografie en Herkomst over van VL-B.
planning najaar 2024.

**Comment by Emilien-Dupont** [comment]
Redactiegroep 17/10/24:
Planning:
ANT: Lieven polst nog bij Lore omtrent het omzetten van het rapport nationaliteit en herkomst wie dit opneemt. 
OVL: hoger- en volwassenenonderwijs (nieuw): voorjaar 2025, Kinderopvang: zomer 2025. Natuur en fossielvrij wonen en bouwen neemt David op zich.

**Comment by Emilien-Dupont** [comment]
- Wanneer het Vlaams Gewest getoond wordt, staat er "Vlaams Gewest" in de tekst en niet "het Vlaams Gewest". Dit zullen we met een get-statement ook kunnen oplossen maar is dat de moeite waard of laten we dat zo? In de rapporten werd dat opgelost met twee lijnen code voor het hele rapport. In Stories zullen we dat dan elke keer moeten doen wanneer we cijfers over het vergelijkingsgebied willen oproepen in een tegel.
- natural breaks in legende als je ze al ziet inde detailview? Er kan eventueel in de glossary aangegeven worden dat je de natural breaks ziet wanneer je op de figuur klikt.

**Comment by Emilien-Dupont** [comment]
RG 18/11/24:
Totdat alle rapporten omgezet zijn, wordt de werkgroep Stories behouden/
TODO Emilien: werkgroep stories samenroepen

**Comment by StefanJacques** [comment]
werkgroep Stories?
vinden jullie dit ook niet storend ?

![image](https://github.com/user-attachments/assets/9997000c-0df4-4818-b193-e0f63fce7436)

Is er een alternatief mogelijk ?

**Comment by Emilien-Dupont** [comment]
Tis inderdaad heel lelijk. Ik heb het al op de agenda geplaatst van de werkgroep volgende week wat we hiermee doen. Ideeën zijn dus altijd welkom ;)

**Comment by Tine-VB** [comment]
Ik stoorde me daar ook aan in de stories van de omgevingsanalyse van Vlaams-Brabant. Ik heb daar dan gewoon de titel van gemaakt: https://vlaamsbrabant.incijfers.be/mosaic/oa-vlaams-brabant/omgevingsanalyse. Maar als we die (lelijke, sorry, mijn mening 😜) blauwe balken willen houden, dan gaat dat natuurlijk niet... Ik vind een banner trouwens ook veel mooier dan een afbeelding.

**Comment by Emilien-Dupont** [comment]
Klopt, maar daar hangt een prijskaartje aan om dat voor elk rapport afzonderlijk te doen. Richard stelde voor om voor dat soort werkzaamheden een strippenkaart af te nemen (komt op de IPWG).

**Comment by Tine-VB** [comment]
In plaats van 'Over dit rapport' kan je er ook gewoon 'Rapport' zetten en dan in de eerste blauwe balk 'Natuur Alken' (of in dit geval misschien beter Natuur in Alken 😁).

**Comment by StefanJacques** [comment]
> In plaats van 'Over dit rapport' kan je er ook gewoon 'Rapport' zetten en dan in de eerste blauwe balk 'Natuur Alken' (of in dit geval misschien beter Natuur in Alken 😁).

mooi alternatief, Tine !!!

**Comment by Tine-VB** [comment]
En de inleiding zou ik er dan gewoon onder zetten, zonder nog eens een titeltje in de linkerbalk.

**Comment by Emilien-Dupont** [comment]
Redactiegroep 21/01/25:
Er is een map in de interne themaboom: index-formule-onderwerpen waarin de onderwerpen geplaatst kunnen worden.
code: vixxxx (zonder index achteraan)
TODO Emilien: toevoegen aan sjabloon stories.
